stats_generation/chronodoses.py

```diff
@@ -0,0 +1,102 @@
+"""Generates stats about «chronodoses».
+
+Issue: https://github.com/CovidTrackerFr/vitemadose/issues/383
+
+Usage:
+
+```shell
+python3 -m stats_generation.chronodoses \
+    --input="some/file.json" # Optional (should follow the same structure as data/output/info_centres.json.)\
+    --output="put/it/here.json" # Optional \
+    --with_national # Optional: Whether or not to add a summary national statistic.
+```
+
+"""
+import argparse
+import json
+import sys
+from pathlib import Path
+from typing import Dict, List
+
+
+_default_input = Path("data", "output", "info_centres.json")
+_default_output = Path("data", "output", "stats_chronodoses.json")
+
+
+def count_departments_chronodoses(data: dict) -> Dict[str, int]:
+    return {
+        department_id: _department_chronodoses(department_data)
+        for department_id, department_data in data.items()
+    }
+
+
+def _with_national(per_department: Dict[str, int]) -> dict:
+    return {
+        "national": sum(per_department.values()),
+        "department": per_department,
+    }
```

Comment:
J'ai un problème avec ton output : il ne se parse pas de la même façon que tu sois en `_with_national` ou sans. Si tu n'as pas `with_national` tu peux itérer dans ton json en t'attendant à avoir des départements, sinon, il faut aller voir dans `department`. 

Je serais pour toujours avoir `department` avec `per_department` dedans, et ajouter `national : ` si `_with_national`.